pkgs/applications/misc/pass-secret-service/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, fetchFromGitHub, python3 }:
+
+with stdenv.lib;
```

Comment:
Don't do that either: https://nix.dev/anti-patterns/language.html#with-attrset-expression

Reply:
Fixed, thanks!